package.json

```diff
@@ -27,6 +28,12 @@
     "add": "npm install && npm --prefix src/server install && npm --prefix admin install",
     "start": "npm run build && NODE_ENV=development webpack-dev-server --host 0.0.0.0 --open --hot --config build/webpack.config.js",
     "build": "npm --prefix admin run build && NODE_ENV=production webpack --config build/webpack.config.js",
-    "serve": "cd src/server && NODE_ENV=development node app.js"
+    "serve": "cd src/server && NODE_ENV=development node app.js",
+    "lint": "eslint ."
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm run lint"
```

Comment:
Can we make it pre-commit?

Reply:
I think pre-commit lint checks would be excessive. Even [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat/blob/0cfb988a82fe80e24d2457f18e97918306184513/package.json#L278) seems to have it on pre-push.

Let me know if you still feel we should do it pre-commit.

Reply:
Knowing linting error after commit will require a dev to either make a new commit or amend that commit. According to me, committing lint error-free code in the first place itself will make life easier for new developers in general who'll contribute to this project as not most of them will know about amending so they'll MP make a new commit.

Reply:
Makes sense! Added the hook.